Ark.Tools.ResourceWatcher/ResourceWatcher.cs

```diff
@@ -352,6 +354,18 @@ private async Task _processEntry(ProcessContext pc, CancellationToken ctk = defa
                 _diagnosticSource.ProcessResourceSaveFailed(info.ResourceId, ex);
             }
         }
+
+        private LocalDateTime? _getModified(IResourceMetadata info)
+        {
+            if (info?.ModifiedMultiple != null && info.ModifiedMultiple.Any())
+            {
+                return info.ModifiedMultiple.Max(x => x.Value);

```

Comment:
seriously? **this is exactly the bug we're trying to change**. 

"Using MAX is bugged, let's use a Dictionary and compare EACH." => use MAX.

Could be a MEME :-D